evap/staff/templates/staff_single_result_form.html

```diff
@@ -11,12 +11,16 @@
 
         <div class="form-group well">
             <div class="col-sm-offset-2 col-sm-6">
-                <input type="submit" value="{% trans "Save course" %}" class="btn btn-primary" onclick="preventAccidentalClosing = false;"/>
+                {% if form.instance.can_staff_edit %}
+                    <input type="submit" value="{% trans "Save course" %}" class="btn btn-primary" onclick="preventAccidentalClosing = false;"/>
+                {% endif %}
```

Comment:
Yep. The same would be nice for staff_course_form.html, but I didn't find a solution for not checking all these ifs again around the div.